tests/test_ec2/test_instances.py

```diff
@@ -50,6 +51,7 @@ def test_instance_launch_and_terminate():
     instances.should.have.length_of(1)
     instances[0].id.should.equal(instance.id)
     instances[0].state.should.equal('running')
+    datetime.datetime.strptime(instances[0].launch_time, '%Y-%m-%dT%H:%M:%S.%f')
```

Comment:
`strptime` will return a `ValueError` if the format doesn't match the string given. See [here](https://docs.python.org/2/library/datetime.html#datetime.datetime.strptime). It seemed like that was sufficient to ensure that `launch_time` was the correct format.